### PR TITLE
Alpha

### PR DESCRIPTION
**Pull request by gnthibault**
Fixed the www capabilities, works quite well now. However there are two problem that remains:
-User are not shown the image directly but must click on the link to jpg (it is not really a big deal though)
-Biggest issue is that the jpg is not a WYSIWYG version of what is shown on the UI. Mainly because of a dynamic issue I presume, Usually resetting the contrast to 0.1 fixes the problem on the jpeg

Other than that it is a pretty cool cool feature, thank you @gehelem  !

### CODE REVIEW

File: als.py [hunk 1]
```diff
@@ -58,14 +58,26 @@ def __init__(self, base_path, server_address, RequestHandlerClass=HTTPHandler):
...
+class StoppableServerThread(threading.Thread):
     """Thread class with a stop() method. The thread itself has to check
     regularly for the stopped() condition."""
 
-    def __init__(self):
-        super(StoppableThread, self).__init__()
+    def __init__(self, web_dir):
+        # web stuff
+        self.web_dir = web_dir
+        self.httpd = HTTPServer(self.web_dir, ("", 8000))
```
Comment (deufrai) [inline als.py]: Would you mind making the port number user configurable, with a default value of 8000 ?

To do that, you'll have to do a rebase of your branch on alpha

Comment (gnthibault) [reply als.py]: Good point @deufrai ! 
I should also take care of the ui counterpart (for the port) + config save. I'll try to tackle this within this week, thank you for the suggestion

Comment (deufrai) [reply als.py]: config is saved just before app quits
you'll just have to add the port number to our config (with a new setter function in config.py)

**Comment by deufrai** [comment]
this PR have been resolved and merged by me locally. Closing

**Pull request closed without merging**